app/lua53/Makefile

```diff
@@ -18,6 +18,20 @@ endif
 
 STD_CFLAGS=-std=gnu11 -Wimplicit -Wall
 
+
+# Validate LUA setting
+ifeq ("$(LUA)","")
+else ifeq ("$(LUA)","53")
+  # ok
+else ifeq ("$(LUA)","51")
+  $(error Your variable LUA="$(LUA)" looks like you probably want \
+    app/lua/Makefile instead)
+else
+  $(error Unsupported value "$(LUA)" for variable "LUA", \
+    expected empty/unset (recommended) or "53")
```

Comment:
Currently only 53 is expected. we will shift the default (on unset/empty) to 53 one day, but not now.

Reply:
I hope the default LUA version for all files inside the `lua53` directory will always be 5.3. If we need to upgrade these files, let's instead rename the directory.

Reply:
This directory should only build if LUA=53 and not when it is empty/unset.
This will change as soon as we shift the default behaviour of the build from 51 to 53, but now it should fail if LUA is unset/empty

Reply:
Oh I see, you want explicit confirmation, not just clarification. That's ok with me.